create-vite-app/template/_App.vue

```diff
@@ -1,10 +1,12 @@
 <template>
-  <h1>Hello Vite + Vue 3!</h1>
-  <p>Edit ./App.vue to test hot module replacement (HMR).</p>
-  <p>
-    <span>Count is: {{ count }}</span>
-    <button @click="count++">increment</button>
-  </p>
+  <div>
```

Comment:
This is not required anymore in vue 3. I think your error comes from your IDE which is not compliant with vue 3 😉

Reply:
@mathieumure Gotcha! Ok so I guess this can be closed. 🙂